llvm/test/CodeGen/DirectX/radians.ll

```diff
@@ -0,0 +1,56 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S  -dxil-op-lower  -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s
```

Comment:
add the scalarizer pass flags to this run line.  Also you will need to rebase and add the ` -dxil-intrinsic-expansion` flag before `-dxil-op-lower` Because a change just went in that decoupled the two.

Reply:
Added scalarizer. Also removed -dxil-op-lower altogether. dxil-intrinsic-expansion is the only thing we want to test here anyway.